packages/firestore/src/api/user_data_converter.ts

```diff
@@ -544,7 +554,8 @@ export class UserDataConverter {
               'of your update data'
           );
         } else {
-          // We shouldn't encounter delete sentinels for queries or non-merge set() calls.
+          // We shouldn't encounter delete sentinels for queries or non-merge
+          // set() calls.
```

Comment:
FYI- You seem to have re-wrapped lines in a few places.  We used to strictly adhere to 80 characters, but I think that enforcement got lost when we went to github and so we've gotten lax...  We probably should have an enforced line length, but until then I'd _slightly_ prefer avoiding reformatting otherwise un-touched lines in PRs. :-)  This PR only has a few and it's not a big deal (so definitely don't bother reverting them or anything).

It seems like we should be able to make prettier enforce a line length, but I haven't looked into it.